### DISCUSSION TITLE
What graphing libraries would you like to see implemented?

### DISCUSSION THREAD

**Discussion by Alek99** [reflex-dev/reflex#195] [Q&A]
So far we have plotly but are looking to add more in the next milestone.

Requirements:
- Must be open source with MIT or fairly open license
     - We are not able to include highcharts because of there restrictive license.

**Comment by Alek99** [comment]
A couple I have in mind:
- Victory
- Nivo

**Comment by FHU-yezi** [comment]
PyEcharts, Python wrapper for Apache Echarts, and it will release v2 after the new year.

**Reply by Alek99** [reply]
Nice ill check it out

**Reply by Aneeqasif** [reply]
any updates on `pyecharts` or `echarts-for-react` integration

**Comment by bkxndata** [comment]
nivo looks really cool.  

just anything besides plotly lol

**Reply by Alek99** [reply]
Yeah nivo is my favorite so far

**Reply by bkxndata** [reply]
curious how nivo does graphing time series graph (i.e. line chart with x axis as datetime objects)

**Reply by Alek99** [reply]
Yeah we are planning to convert python date time to js date if used

**Comment by gsal** [comment]
bokeh ?

**Comment by simonmesmith** [comment]
Late to the party, but I'd really appreciate a directed graph option (as mentioned in #1245).

**Comment by rraammiinn** [comment]
nextui

**Reply by rraammiinn** [reply]
https://nextui.org

**Reply by Alek99** [reply]
From what I see this is more of a component library rather than a graphing lib is there a specific graph you want from here?

**Reply by rraammiinn** [reply]
I mean it's good to have more component libraries .
or maybe better to have some sort of cli tool for adding any arbitrary component library to pynecone project .
